relay-event-normalization/src/normalize/span/exclusive_time.rs

```diff
@@ -42,6 +42,11 @@ fn set_event_exclusive_time(
         return;
     };
 
+    if trace_context.exclusive_time.value().is_some() {
+        // Exclusive time already set, respect.
```

Comment:
I respect that, bro 🤣 

```suggestion
        // Exclusive time already set, respect it.
```